### PR TITLE
Documentation cleanup

### PR DESCRIPTION
**Pull request by smhutch**
## What kind of change does this PR introduce?

Minor docs updates

## Checklist

- [x] Documentation
- [x] Testing 
- [x] Ready to be merged
      
- [ ] Added myself to contributors table

### CODE REVIEW

File: packages/app/src/app/pages/Sandbox/Editor/Workspace/screens/GitHub/CreateRepo.tsx [hunk 1]
```diff
@@ -42,7 +42,7 @@ export const CreateRepo = () => {
       <Element paddingX={2}>
         <Text variant="muted" marginBottom={4} block>
           Export the content of this sandbox to a new GitHub repository,
-          allowing you to commit changes made on Codesandbox to GitHub. If you
+          allowing you to commit changes made on CodeSandbox to GitHub. If you
```
Comment (smhutch) [inline packages/app/src/app/pages/Sandbox/Editor/Workspace/screens/GitHub/CreateRepo.tsx:45]: I changed all occurences of `Codesandbox` spacing, apart from those in the old privacy policy (which should not be changed).

File: packages/homepage/content/docs/1-x-faqs.md [hunk 2]
```diff
@@ -83,13 +83,16 @@ from File > Preferences > Settings in the editor.
...
-  read to and from them in your code code and they can be seen and edited via
+  read to and from them in your code and they can be seen and edited via
   the terminal.
 - Terminal commands which alter the filesystem of the container instance aren't
   synced with files shown in the editor. You'll need to refresh to see files
   updated this way.
 - Container sandboxes sleep after around 10 minutes and can be woken by opening
   the sandbox or preview in a web browser.
+- Container sandboxes have a 1GB persistent storage limit, a 1GB vCPU soft limit,
+  and a hard memory limit of 2GB.
```
Comment (smhutch) [inline packages/homepage/content/docs/1-x-faqs.md:94]: This comes up a few times with large container sandboxes. Better to have it mentioned in the docs.

**Comment by lbogdan** [comment]
Build for latest commit https://github.com/codesandbox/codesandbox-client/pull/5812/commits/cb87cd3e7cddbdba67d3b0056273645f30bf26f6 is at https://pr5812.build.csb.dev/s/new.

**Comment by smhutch** [comment]
@CompuIves could you give this a quick 👀 to see that everything makes sense please?

File: packages/app/src/app/pages/Sandbox/Editor/Workspace/screens/GitHub/CreateRepo.tsx (hunk 1 above)
Comment (CompuIves) [reply packages/app/src/app/pages/Sandbox/Editor/Workspace/screens/GitHub/CreateRepo.tsx:45]: Great!!

File: packages/homepage/content/docs/1-x-faqs.md (hunk 2 above)
Comment (CompuIves) [reply packages/homepage/content/docs/1-x-faqs.md:94]: Great, looks good to me. What do you think @lbogdan ?

**Review by CompuIves — APPROVED** [review]
This is very good! Much better & professional 👍

**Pull request merged**